File: Server/src/main/java/bgu/spl171/net/impl/rci/LOGRQ.java
package bgu.spl171.net.impl.rci;

public class LOGRQ extends FileNameMessage {

	String userName;
	
	public LOGRQ(){
		opcode = 7;
	}

	public String getUserName() {
		return userName;
	}

	public void setUserName(String userName) {
		this.userName = userName;
	}
	
	
}

File: Client/src/Messages/LOGRQ.cpp
#include "FileNameMessage.cpp"
#include <string>
using namespace std;
 class LOGRQ :public FileNameMessage {

	string userName;
  string messageType = "LOGRQ";

	public:
     string getUserName() {
		return userName;
	}

	 void setUserName(string userName) {
		this->userName = userName;
	}


};
